### ISSUE TITLE
Can I increase size of a Detected Plane dynamically?

### ISSUE THREAD

**Issue by deeivid** [google-ar/arcore-unity-sdk#335]
Lets say I have a huge wall but I do not want to physically move around with my telephone to detect all the wall as a plane.

Can I scan a small area of the wall, to detect a small plane, and then increase the size of this plane dinamically so more area of the wall is mapped as a detected plane?

Thank you.

**Comment by iBicha** [comment]
Check the example scene, planes expand and gets merged as they are detected

**Comment by deeivid** [comment]
Yes, it does this _automatically_ if I physically move my phone around. But what if I want to "_manually_" expand and already detected plane. Is this possible?

Thanks!

**Comment by pablisho** [comment]
Hi, DetectedPlane size will always follow what ARCore has detected. If you want to expand it as you wish, you'll have to do it in the visualization of it. In HelloAR we're using [DetectedPlaneVisualizer](https://github.com/google-ar/arcore-unity-sdk/blob/master/Assets/GoogleARCore/Examples/Common/Scripts/DetectedPlaneVisualizer.cs). Let me know if this answers your question.

**Comment by deeivid** [comment]
Dynamically increasing the visualization area of the detected plane is as good as having ARCore detect and automatically increase the plane?

Thanks for the help.

**Comment by pablisho** [comment]
ARCore won't do that, so increasing the visualization is your only option I believe.
